main.tf

```diff
@@ -732,3 +732,34 @@ module "magnum" {
     "cluster-user-trust" = "true"
   }
 }
+
+resource "juju_application" "ldap-apps" {
+  #  Transform list of maps into a map of maps
```

Comment:
nit: ldap_apps is already map of maps. so the comment in l#737 might be misleading